src/scores/probability/brier_impl.py

```diff
@@ -61,3 +67,155 @@ def brier_score(
         check_binary(obs, "obs")
 
     return mse(fcst, obs, reduce_dims=reduce_dims, preserve_dims=preserve_dims, weights=weights)
+
+
+def ensemble_brier_score(
+    fcst: XarrayLike,  # type: ignore
+    obs: XarrayLike,  # type: ignore
+    ensemble_member_dim: str,
+    thresholds: Union[float, int, Sequence[float]],
+    *,  # Force keywords arguments to be keyword-only
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    weights: Optional[xr.DataArray] = None,
+    fair_correction: bool = True,
+    threshold_mode: Literal[">=", ">"] = ">=",
```

Comment:
Should this be Set(operator.ge, operator.gt)? It's better to use function pointers than logic based on string matching, there are far too many errors than can be introduced through interpreting strings.

Reply:
Sure - I'll make an issue to update the binary discretise function that this is calling https://github.com/nicholasloveday/scores/blob/1485879a1b8171de7ca89cb124680c60c6cc198d/src/scores/processing/discretise.py#L97